src/assets/js/boot-options.mjs

```diff
@@ -1,5 +1,55 @@
-'use strict';
+/* global browser */
 
-import {initOptions} from "@/options.mjs";
+import ReactDOM from 'react-dom/client';
+import React from 'react';
+import { useEffect } from 'react';
+import {
+  Provider,
+  useSelector,
+  useDispatch,
+} from 'react-redux';
 
-initOptions();
+import 'bootstrap/scss/bootstrap.scss';
+import '@/scss/style.scss';
+
+import { SignIn } from '@/components/options/sign-in.jsx';
+import { SignedIn } from '@/components/options/signed-in.jsx';
+
+import { store } from '@/store.js';
+import { signIn } from '@/features/auth/authSlice.js';
+
+const Options = () => {
+  const signedIn = useSelector((state) => state.auth.signedIn);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    browser.storage.sync.get('token').then((result) => {
+      if (result.token) {
+        dispatch(signIn());
+      }
+    });
+  }, []);
+
+  return (
+    <div className="container">
+      <div className='container container-small'>
```

Comment:
For attributes within tags, prefer using double quotes everywhere. For any JS stuff use single quotes.